docs/source/overview.rst

```diff
@@ -71,3 +71,8 @@ ElasticBLAST performs the searches with the BLAST+ package, and most of the BLAS
 
 To do your first ElasticBLAST run, go to the :ref:`quickstart-gcp` or the :ref:`quickstart-aws`
 
+**Are Cloud Credits available?**
+
+AWS and GCP have programs to award Cloud Credits to researchers at accredited research institutions.  You can read about the AWS program `here <https://aws.amazon.com/government-education/research-and-technical-computing/cloud-credit-for-research/>`_ and the GCP program `here <https://edu.google.com/programs/credits/research/?modal_active=none>`_.
+
+AWS and GCP also have more limited Free Tiers.  Read about the AWS one `here <https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all>`_ and the GCP one `here <https://cloud.google.com/free>`_.  While these are a good way to learn about the cloud, they may be too limited for you to effectively use ElasticBLAST.
```

Comment:
The link to the AWS Free Tier information is needlessly long. The one proposed below is a better alternative IMHO.

```suggestion
AWS and GCP also have more limited Free Tiers.  Read about the AWS one `here <https://aws.amazon.com/free>`_ and the GCP one `here <https://cloud.google.com/free>`_.  While these are a good way to learn about the cloud, they may be too limited for you to effectively use ElasticBLAST.
```